Packs/FortiSIEM/IncidentFields/incidentfield-FortiSIEM_Destination_User.json

```diff
@@ -0,0 +1,29 @@
+{
+    "associatedToAll": false,
+    "associatedTypes": [
+        "FortiSIEM"
+    ],
+    "caseInsensitive": true,
+    "cliName": "destinationuser",
+    "closeForm": false,
+    "content": true,
+    "editForm": true,
+    "group": 0,
+    "hidden": false,
+    "id": "incident_destinationuser",
```

Comment:
Please add prefix to id.
should be `incident_fortisiemdestinationuser`

Reply:
Done.